themes/default/layouts/index.html

```diff
@@ -243,27 +190,28 @@ <h6>{{ $item.title }}</h6>
         </div>
     </section>
 
-    <!-- Logos -->
-    <section class="container mx-auto text-center my-0 mb-16 lg:my-24 overflow-hidden px-3">
-        <div class="mb-16">
-            <h3>Atlassian reduced time spent on maintenance by 50%</h3>
-            <div class="max-w-2xl mx-auto text-center">
-                <div class="max-w-xs mx-auto">
-                    {{ partial "customer-logo.html" (dict "logo" "atlassian") }}
-                </div>
-                <p class="mt-0">
-                    "Using Pulumi and Python, our team quickly and easily built a repeatable pipeline that deploys development environments for more than 100 team members
-                    worldwide. The benefits of working in languages we know can’t be overstated. That just made moving to the Pulumi Cloud Engineering Platform an obvious choice
-                    for the team."
+    <section class="container mx-auto my-16 text-center">
+        <h2 class="rainbow-text">Accomplish more without increasing headcount</h2>
```

Comment:
I feel like the word "headcount" is too business value focused and not speaking enough to the developer and engineer. I want to tell the developer what they are directly getting with Automation API